src/screens/AddNotificationEmailScreen.tsx

```diff
@@ -96,8 +96,8 @@ class AddNotificationEmailScreen extends PureComponent<Props, State> {
     }
     checkSubscription(wallets, email, {
       onSuccess: (ids: string[]) => {
-        const walletsToSubscribe = wallets.filter(w => !ids.includes(w.id));
-        if (!walletsToSubscribe.length) {
+        const walletsToSubcribe = wallets.filter(w => !ids.includes(w.id));
+        if (!wallets.length) {
```

Comment:
this 'if' can be at the very top of the method

Reply:
in fact here should be `if (! walletsToSubcribe.length) {` should be here

Reply:
done